src/Report/EnabledMembers.php

```diff
@@ -0,0 +1,206 @@
+
+    public function title()
+    {
+        return _t(__CLASS__ . '.TITLE', parent::title());
+    }
+
+    public function description()
+    {
+        return _t(__CLASS__ . '.DESCRIPTION', parent::description());
+    }
+
+    /**
+     * @param array $params
+     */
+    public function sourceRecords($params)
+    {
+        $methods = $this->getMethodClassToTitleMapping();
+
+        // Any GridFieldAction will submit all form inputs, regardless of whether or not they have valid values.
+        // This results in zero results when no filters have been set, as the query then filters for empty string
+        // on all parameters. `stlen` is a safe function, as all user input comes through to PHP as strings.
+        $params = array_filter($params, 'strlen');
+
+        // Having a WHERE limits the results, which means the the COUNT will be off (GROUP BY applied after WHERE)
+        // So clip the count filter and apply it after the database query, only if Methods filter is also set
+        // this way we can delegate to the database where possible, which should be faster.
+        if (isset($params['Count'])) {
+            $desiredCount = (int)$params['Count'];
+            if (isset($params['Methods'])) {
+                $countFilter = $desiredCount;
+                unset($params['Count']);
+            } elseif ($desiredCount) {
+                // When querying SQL COUNT() - this will always factor in the backup method, which is excluded in the
+                // secondary query below to get the list of method names the user has registered. So to compensate we
+                // need to bump up the entered number by 1.
+                $params['Count'] = $desiredCount + 1;
+            }
+        }
+
+        $filteredList = ArrayList::create([]);
+        $backupClass = get_class(singleton(MethodRegistry::class)->getBackupMethod());
+        /** @var Member[]&MemberExtension[] $members */
+        $members = $this->applyParams(Member::get(), $params)->toArray();
+        foreach ($members as $member) {
+            $defaultMethod = $member->getDefaultRegisteredMethod();
+            $defaultMethodClassName = $defaultMethod ? $defaultMethod->MethodClassName : '';
+
+            $registeredMethods = $member
+                ->RegisteredMFAMethods()
+                ->exclude('MethodClassName', $backupClass)
+                ->column('MethodClassName');
+            $registeredMethodNames = array_map(function (string $methodClass) use ($methods) {
+                return $methods[$methodClass];
+            }, $registeredMethods);
+
+            $memberReportData = ArrayData::create();
+            foreach ($member->summaryFields() as $field => $name) {
+                $memberReportData->$field = $member->$field;
+            }
+
+            $memberReportData->methodCount = (string)(count($registeredMethods) ?: '');
+            $memberReportData->registeredMethods = implode(', ', $registeredMethodNames);
```

Comment:
The combination of these two columns takes up about 40-50% of the width of the screen for me. I'm not convinced you need the "Number of registered methods" columns, since the "Registered methods" column indicates that pretty clearly anyway. Thoughts?

Reply:
Similar to your earlier comment about cognitive load digesting a line of code, I find having a simple number easier to read than having to process a list of method names.
Super keen on alternate naming suggestions though :)

Reply:
What about appending the number of methods to the list of methods, eg:

`Authenticator app, Security key (2 methods)`

Reply:
Maybe it's worth showing @silverstripeux?

Reply:
>What about appending the number of methods to the list of methods, eg:
>
>`Authenticator app, Security key (2 methods)`

Then it becomes extremely difficult to filter in an export.

Reply:
If you want to keep the number column, you could call it `No. methods` which would reduce the width taken up

Reply:
Used unicode numero. Changed names column to state that it is names.